view/frontend/web/js/view/promo/promo-display.js

```diff
@@ -4,15 +4,14 @@ define([
 ], function ($, amazonCheckout) {
     'use strict';
 
-
-
     return function (config) {
 
         $(document).ready(function () {
 
             amazonCheckout.withAmazonCheckout(function (amazon) {
 
-                amazon.Pay.renderPromotionalMessage('.ap-promotional-message', {
+                const selector = ('.' + config.bannerClass);
```

Comment:
With the other changes this shouldn't be necessary.

Reply:
i'm passing the class to the element in the template, as well as in the config args so i think i need this here as is, but if you had somethin else in mind let me know!

Reply:
I was thinking of it as `config.bannerClass` being the complete selector — If you want to keep it separate then I would probably clean this up to something like ```(`.${config.bannerClass}`, ...```

Reply:
so like this in the template? "bannerClass": "<?= $escaper->escapeHtml('.' . $promoBannerClass) ?>"?

Reply:
Yea, we should probably reference it as `bannerSelector` if you make that change.